openedx_lti_tool_plugin/tests/test_utils.py

```diff
@@ -23,35 +24,89 @@ def setUp(self):
         super().setUp()
         self.request_mock = MagicMock()
 
-    def test_with_available_sequences(
+    def test_with_available_sequences_and_available_units(
         self,
         timezone_mock: MagicMock,
         datetime_mock: MagicMock,
         get_user_course_outline_mock: MagicMock,
         get_outline_tree_mock: MagicMock,
-        from_string_mock: MagicMock,
+        course_key_from_string_mock: MagicMock,
+        modulestore_mock: MagicMock,
     ):
-        """Test get_course_outline with available sequences.
+        """Test get_course_outline with available sequences and available units.
 
         Args:
             timezone_mock: Mocked timezone function.
             datetime_mock: Mocked datetime function.
             get_user_course_outline_mock: Mocked get_user_course_outline function.
             get_outline_tree_mock: Mocked get_course_outline_block_tree function.
-            from_string_mock: Mocked CourseKey from_string method.
+            course_key_from_string_mock: Mocked CourseKey from_string method.
+            modulestore_mock: Mocked modulestore class.
         """
-        get_user_course_outline_mock.return_value = MagicMock(sequences=['test'])
+        get_user_course_outline_mock.return_value = MagicMock(sequences=['test-sequence'])
+        modulestore_mock.get_items.return_value = [MagicMock(location='test-unit')]
 
-        self.assertEqual(get_course_outline(self.request_mock, COURSE_ID), COURSE_BLOCKS)
-        from_string_mock.assert_called_once_with(COURSE_ID)
+        outline = get_course_outline(self.request_mock, COURSE_ID)
+
+        self.assertEqual(outline, COURSE_BLOCKS)
```

Comment:
```suggestion

        self.assertEqual(get_course_outline(self.request_mock, COURSE_ID), COURSE_BLOCKS)
```
@alexjmpb What do you think about assert the call without defining the variable?